awx/main/access.py

```diff
@@ -1325,14 +1325,14 @@ class ExecutionEnvironmentAccess(BaseAccess):
 
     def filtered_queryset(self):
         return ExecutionEnvironment.objects.filter(
-            Q(organization__in=Organization.accessible_pk_qs(self.user, 'execution_environment_admin_role')) |
+            Q(organization__in=Organization.accessible_pk_qs(self.user, 'member_role')) |
             Q(organization__isnull=True)
         ).distinct()
 
     @check_superuser
     def can_add(self, data):
         if not data:  # So the browseable API will work
-            return Organization.accessible_objects(self.user, 'execution_environment_admin_role').exists()
+            return Organization.accessible_objects(self.user, 'member_role').exists()
```

Comment:
I _think_ that this change isn't correct, but am willing to be shown wrong.

Reply:
I agree this looks like any member can create one

Reply:
it sure does, and I tested it just to verify. I'll pull it out